common/app/common/TrailsToShowcase.scala

```diff
@@ -106,7 +121,7 @@ object TrailsToShowcase {
       )
     }
 
-    (rundownPanelOutcome, singleStoryPanelsOutcome)
+    (rundownPanelOutcome, singleStoryPanelsOutcome, duplicateMap)
```

Comment:
Could we keep the same name across the codebase, as it's a triplet it'll make it easer to refactor / fathom.

Reply:
Thanks James - have named it `duplicateMap` throughout now. Are we good to merge in the A.M.?